sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringProjectsClient.cs

```diff
@@ -48,7 +48,7 @@ public QuestionAnsweringProjectsClient(Uri endpoint, AzureKeyCredential credenti
             Argument.AssertNotNull(credential, nameof(credential));
             options ??= new QuestionAnsweringClientOptions();
 
-            ClientDiagnostics = new ClientDiagnostics(options);
+            ClientDiagnostics = new ClientDiagnostics(options, true);
```

Comment:
@heaths FYI, any concerns if we suppress nested distributed tracing spans for QA?  We are doing this for all DPGs.

Reply:
I don't think so.